### ISSUE TITLE
RSS feed

### ISSUE THREAD

**Issue by madstk1** [arndom/devy-brew#1]
Hello,

Would it fit the project's model to provide an RSS-feed?
I know a handful of people, myself included, would like this feature.

**Comment by arndom** [comment]
Yeah that can be added, though I haven't implemented before, it shouldn't be too hard to do
